app/workers/education_form/forms/va_5490.rb

```diff
@@ -65,5 +65,25 @@ def previous_benefits
 
       previous_benefits_arr.join("\n")
     end
+
+    def veteran_date_of_death_label
+      if @applicant.benefit != 'chapter33' || !Flipper.enabled?(:edu_form5490_updates) || @applicant.sponsorStatus.nil?
+        return 'Date of death or MIA/POW:'
+      end
+
+      label = ''
+
+      case @applicant.sponsorStatus
```

Comment:
label variable isn't needed, you can just have the string returned after the case is evaluated. If you'd like to keep the variable however, another acceptable method is instead of instantiating label as an empty string, you could set it equal to the evaluation of `case @applicant.sponsorStatus`

Reply:
Thanks! I just assumed rubocop would complain.

Reply:
No worries, everything else looks good